web/modules/weather_data/src/Service/WeatherDataService.php

```diff
@@ -105,6 +105,12 @@ public function __construct(
      */
     public function getFromWeatherAPI($url, $attempt = 1, $delay = 75)
     {
+        if (!str_starts_with($url, "https://")) {
+            $baseUrl = getEnv("API_URL");
+            $baseUrl = $baseUrl == false ? "https://api.weather.gov" : $baseUrl;
+            $url = $baseUrl . $url;
+        }
```

Comment:
Prepends the server URL. This is where we can inject the URL from environment variables.